posthog/batch_exports/models.py

```diff
@@ -292,4 +292,10 @@ def workflow_id(self) -> str:
         end_at = self.end_at and self.end_at.isoformat()
         start_at = self.start_at and self.start_at.isoformat()
 
+        if isinstance(end_at, str) and end_at.endswith("+00:00"):
+            end_at = end_at.replace("+00:00", "Z")
```

Comment:
looks fine, but just wondering if there's there a reason you can't just do

```python
end_at = self.end_at and self.end_at.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Reply:
I actually reverted this and copied the same formatting we use here: https://github.com/PostHog/posthog/blob/master/posthog/batch_exports/service.py#L475

Makes sense to have the service and model be consistent. But we could eventually update both.